app/models/distribution.rb

```diff
@@ -29,22 +29,20 @@ class Distribution < ApplicationRecord
   # Distributions contain many different items
   include Itemizable
   include Exportable
+  include IssuedAt
+  include Filterable
 
   has_one :request, dependent: :nullify
   accepts_nested_attributes_for :request
 
   validates :storage_location, :partner, :organization, :delivery_method, presence: true
   validate :line_item_items_exist_in_inventory
 
-  include IssuedAt
-
   before_save :combine_distribution
 
   enum state: { started: 0, scheduled: 5, complete: 10 }
-
   enum delivery_method: { pick_up: 0, delivery: 1 }
 
-  include Filterable
```

Comment:
It's OK to move this up to the top if you think it's easier to read -- I believe these were placed there because the scopes below it are teh scopes used in the filters (in case you were wondering why it was spread out)

Reply:
@armahillo Got it! I can move it back if you want! I was following this style guideline from Rubocop, but it isn't enforced for the project: https://github.com/rubocop-hq/ruby-style-guide#consistent-classes